### PR TITLE
Highlight printed function differently.

### PR DESCRIPTION
**Pull request by jviereck**
I don't like that the printed functions are not distinguishable from the executed input. Therefore, I make the printed functions get highlighted a little bit. 

You can see the output here: http://postimage.org/image/i12a5svlp/

### CODE REVIEW

File: chrome/jsterm.js [hunk 1]
```diff
@@ -164,21 +164,22 @@ let JSTermUI = {
...
-      this.output.setText(startWith + msg, this.output.getCharCount());
+      let out = startWith + msg;
+      this.output.setText(out, this.output.getCharCount());
       let line = this.output.getLineCount() - 1;
-      this.objects.set(line, object);
-      this.markRange(line);
-
+      if (isAnObject)
+        this.objects.set(line, object);
+      this.markRange(line, isAnObject, out.split('\n').length - 1);
```
Comment (jviereck) [inline chrome/jsterm.js:231]: Any better idea to get the number of lines then using `.split('\n')`?

Comment (blissdev) [inline chrome/jsterm.js:231]: I imagine you could use a regular expression and count the matches.

**Comment by blissdev** [comment]
I like what you did here, but I'm wondering if it would be possible to add padding to the printed functions?

**Comment by paulmillr** [comment]
Would you rebase this with current master please?

**Comment by jviereck** [comment]
@paulmillr would you merge it :) ?

**Comment by paulmillr** [comment]
sure

**Pull request closed without merging**